courses-fe/src/pages/login/LoginComponent.js

```diff
@@ -9,15 +9,15 @@ const LoginPage = ({loginUser}) => {
     const [password, setPassword] = useState('');
     const [submitted, setSubmitted] = useState(false);
 
-    const handleSubmit = (event) => {
-        event.preventDefault();
-        setSubmitted(true);
-        if (username && password) {
-            navigate("/home");
-        } else {
-            console.log('error');
-        }
-    }
+    // const handleSubmit = (event) => {
```

Comment:
daca ai comentat ceva mai bine stergi de tot...ca altfel se creaza smelyy code...adica cine vine si citeste codu nu stie daca trebuie sau nu sa lase comentariul si face si el mai departe..si o sa ajugem sa aavem o gramade de commenturi peste tot aiurea. Comenturile trebuie puse doar temporar si doaar daca ai ceva de zis despre o bucata de cod...nu pui in comment codul in sine. Deci daca l-ai cumentat atunci mai bine il stergi ca inseamnaa ca nu trebuie folosit.